cmd/lncli/main.go

```diff
@@ -563,3 +566,74 @@ func networkParams(ctx *cli.Context) (*chaincfg.Params, error) {
 		return nil, fmt.Errorf("unknown network: %v", network)
 	}
 }
+
+func waitUntilStatus(ctx *cli.Context, desiredState lnrpc.WalletState,
+	timeout time.Duration) error {
+	connectionRetryTicker := time.NewTicker(connectionRetryInterval)
```

Comment:
nit: after a multi-line function header, there should be an empty line before code starts.